cdatweb/settings/development.py

```diff
@@ -20,8 +20,8 @@
 
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 
-VISUALIZATION_LAUNCHER = 'http://aims1.llnl.gov/vtk'
-#VISUALIZATION_LAUNCHER = 'http://localhost:7000/vtk'
+#VISUALIZATION_LAUNCHER = 'http://aims1.llnl.gov/vtk'
+VISUALIZATION_LAUNCHER = 'http://localhost:7000/vtk'
```

Comment:
I think it was desired to point to the aims1 server by default.  I don't really care either way.  If we use aims1, then we should probably fix the README as well.